code/game/objects/items/weapons/RPD.dm

```diff
@@ -168,6 +168,12 @@ var/global/list/RPD_recipes=list(
 /obj/item/weapon/pipe_dispenser/attack_self(mob/user)
 	show_menu(user)
 
+/obj/item/weapon/pipe_dispenser/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] points the end of the RPD down \his throat and presses a button! It looks like \he's trying to commit suicide...</span>")
```

Comment:
An atmos pipe has a height of 1m and a radius of approximately 15cm
In freedom units that's a height of about 1 yard and a diameter of about 1 foot
You don't want that shit inside of you